workspaces/default/themes/base/partials/footer.html

```diff
@@ -5,7 +5,8 @@
         <h1 class="brand">{{portal.name}}</h1>
       </div>
 
-      <nav class="eight columns">
+      <nav class="eight columns" aria-labelledby="footer-nav">
+        <div id="footer-nav" hidden>Footer Links</div>
```

Comment:
Do we have to do `aria-labelledby` here with a hidden element or can we just do `aria-label="Footer Links"`?